internal/pkg/service/buffer/api/service/service.go

```diff
@@ -559,6 +559,9 @@ func setupMappingTable(ctx context.Context, client client.Sender, mapping model.
 
 func generateExportToken(ctx context.Context, client client.Sender, export *model.Export) (err error) {
 	token, err := storageapi.CreateTokenRequest(
+		storageapi.WithDescription(
+			fmt.Sprintf("[_internal] Buffer Export %s for Receiver %s", export.ReceiverKey.ReceiverID, export.ExportID),
+		),
```

Comment:
Overis prosim v #team_connection maximalnu dlzku `name` pri tokene a ze to splname?
Receiver a export ID maju definovanu maximalnu dlzku.

Mozno nie je potrebna ziadna zmena v kode, iba komentar, ze maximalna dlzka je napr. `255` a dany string ma maximalne napr. `210` znakov.